Lib/pdb.py

```diff
@@ -79,6 +79,7 @@
 import codeop
 import pprint
 import signal
+import asyncio
```

Comment:
Line 614 has deferred import of `asyncio.base_futures`, so we could consider adding a lazy import of `asyncio` at line 606 to improve import time?

https://github.com/python/cpython/issues/109653
https://github.com/python/cpython/issues/118761

Reply:
asyncio will be used as long as pdb is used. `pdb` is not an module that is often imported explicitly or implicitly, I don't think import time of `pdb` is specifically critical.

Reply:
You're right. I didn't realize that at the time